src/state/reducer.js

```diff
@@ -266,14 +266,13 @@ export default (state: State = clean('IDLE'), action: Action): State => {
         return existing;
       }
 
-      // $ExpectError - using spread
```

Comment:
By casting to `any` are we loosing some type safety? Also, the $ExpectError is very clear about why the following statement fails the check. I think just casting to `any` looses that context. Additionally, hopefully in the future we can remove the ExpectError if flow can understand Object spread a little better. Without further information I think we should move back our spread usage to using ExpectError rather than any.

Reply:
This goes for all of these types of changes

Reply:
Perhaps we could use [$Exact<>](https://flow.org/en/docs/types/utilities/#exactt-a-classtoc-idtoc-exact-hreftoc-exacta) here

Reply:
`$ExpectError` is not defined by default, leading to unfixable flow errors for consumers. Should be `$FlowFixMe` if you want to ignore it.

I'd recommend removing the custom config here: https://github.com/atlassian/react-beautiful-dnd/blob/4f4c3307a4e0bf066cf8e7dea694d40d8aac9aab/.flowconfig#L10

(`$FlowFixMe` works out of the box)

Reply:
The problem is that flow consider all spreads as inexact. We are loosing the same safety with suppress comment too, but it's more explicit and won't hide different kinds of errors.

Reply:
$ExpectError should be used only in flow tests.